mslib/msui/topview.py

```diff
@@ -185,13 +185,27 @@ class MSUITopViewWindow(MSUIMplViewWindow, ui.Ui_TopViewWindow):
     signal_permission_revoked = QtCore.pyqtSignal(int)
     signal_render_new_permission = QtCore.pyqtSignal(int, str)
 
-    def __init__(self, parent=None, model=None, _id=None, active_flighttrack=None, mscolab_server_url=None, token=None):
+    def __init__(self, parent=None, mainwindow=None, model=None, _id=None,
+                 active_flighttrack=None, mscolab_server_url=None, token=None):
         """
         Set up user interface, connect signal/slots.
         """
         super().__init__(parent, model, _id)
         logging.debug(_id)
-        self.ui = parent
```

Comment:
wouldn't a reference to mainwindow not "simpler" here?

Reply:
Sideview and Linearview don't have multiple flightpath dockingwidget. By this the problem was introduced.